emcc.py

```diff
@@ -1531,15 +1531,17 @@ def is_supported_link_flag(f):
     if shared.Settings.EXPORT_ES6 and not shared.Settings.MODULARIZE:
       exit_with_error('EXPORT_ES6 requires MODULARIZE to be set')
 
-    # When MODULARIZE option is used, currently declare all module exports
-    # individually - TODO: this could be optimized
     if shared.Settings.MODULARIZE and not shared.Settings.DECLARE_ASM_MODULE_EXPORTS:
-      shared.Settings.DECLARE_ASM_MODULE_EXPORTS = 1
-      shared.warning('Enabling -s DECLARE_ASM_MODULE_EXPORTS=1, since MODULARIZE currently requires declaring asm.js/wasm module exports in full')
-
-    # In MINIMAL_RUNTIME when modularizing, by default output asm.js module under the same name as the JS module. This allows code to share same loading function for both JS and asm.js modules,
-    # to save code size. The intent is that loader code captures the function variable from global scope to XHR loader local scope when it finishes loading, to avoid polluting global JS scope with
-    # variables. This provides safety via encapsulation. See src/shell_minimal_runtime.html for an example.
+      # When MODULARIZE option is used, currently requires declaring all module exports
+      # individually - TODO: this could be optimized
+      shared.warning('DECLARE_ASM_MODULE_EXPORTS=0 is not compatible with MODULARIZE')
```

Comment:
this just warns, but continues onward (without erroring of changing the values)?

Reply:
oops. thanks.